### PR TITLE
Add sponsors to Cloud Engineering Summit landing page

### PR DESCRIPTION
**Pull request by zchase**
This PR adds the logos of the sponsors for the Cloud Engineering Summit and adds a link to navigate back to the landing page from the session list.

### CODE REVIEW

File: themes/default/layouts/page/cloud-engineering-summit-session-list.html [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 {{ define "main" }}
     <section id="sessions" class="container mx-auto my-16 text-center">
+        <div class="text-left mb-12">
+            <a class="text-blue-600 hover:underline" href="{{ relref . "/cloud-engineering-summit" }}">&#8592; Go back to the Cloud Engineering Summit home page</a>
```
Comment (susanev) [inline themes/default/layouts/page/cloud-engineering-summit-session-list.html:8]:
```suggestion
            <a class="text-blue-600 hover:underline" href="{{ relref . "/cloud-engineering-summit" }}">&#8592; Cloud Engineering Summit</a>
```

File: themes/default/layouts/page/cloud-engineering-summit.html [hunk 2]
```diff
@@ -96,6 +96,39 @@ <h3>{{ $item.name }}</h3>
...
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/honeycomb-logo.svg" alt="Honeycomb" />
+            </div>
+
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/nginx.svg" alt="NGINX" />
+            </div>
+
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/ns1.png" alt="NS1" />
```
Comment (susanev) [inline themes/default/layouts/page/cloud-engineering-summit.html:127]: not blocking, but this logo is getting significantly more weight than the others;if its easy to reduce its size a bit compared to the others and this wasn't intentional than id recommend fixing it

**Review by susanev — APPROVED** [review]

**Review by cnunciato — APPROVED** [review]
:shipit:

**Comment by isaac-pulumi** [comment]
Please hold this for sponsor approval (in preview).

**Review by isaac-pulumi — APPROVED** [review]

**Pull request merged**